cluster/config-defaults.yaml

```diff
@@ -204,7 +204,7 @@ cluster_dns: "coredns"
 coredns_log_svc_names: "true"
 
 coreos_image: "ami-06142be4afc3d5b1b" # Container Linux 2079.4.0 (HVM, eu-central-1)
-kuberuntu_image: "ami-07922c3ba50e085fb" # Kuberuntu (dev) (HVM, eu-central-1)
+kuberuntu_image: "ami-064dbf2c30178e807" # Kuberuntu (dev) (HVM, eu-central-1)
```

Comment:
Could we please use the latest AMI > ami-0d856c4c2daf9b569, so see if the boot process has improved?

Reply:
I would defer this to later as it would take another 1h to run e2e 😞

Reply:
Then let's merge it